eng/common/template-guidance.md

```diff
@@ -57,7 +57,7 @@ extends:
 
 Note: Multiple outputs are ONLY applicable to 1ES PT publishing (only usable when referencing `templates-official`).
 
-## Development notes
+# Development notes
```

Comment:
@radical - I thought you fixed this. What happened here?

Reply:
Looks like it wasn't backported to the 9.0 branch.

Reply:
Can you backport it?